packages/cli/migrations/migration-collection.json

```diff
@@ -30,6 +30,11 @@
             "version": "5.0.3",
             "description": "Updates Ignite UI for Angular project from CLI 5.0.x to 5.0.3",
             "factory": "./update-5_0_3"
+        },
+        "migration-07": {
```

Comment:
Hrm, the situation w/ migrations in the CLI is a bit odd. Keep in mind this will only run for `ng update igniteui-cli` which is not necessarily present in all projects - the rest should use the schematics package only, which is the preferred method ATM anyway.

Not sure, but I believe the schematics should be grouped with the CLI main package for migrations (check package.json) so you should be able to write a migration there that can apply in both cases.

Reply:
On second thought, not sure how wise it is to modify existing projects' scrollbars. While I think it'd be an improvement, it could be undesirable for some and it's always iffy to do that on existing project

Reply:
migration was reverted now